packages/frontend-shared/src/components/Button.vue

```diff
@@ -126,7 +126,11 @@ const props = defineProps<{
 
 const attrs = useAttrs() as ButtonHTMLAttributes
 
-const variantClasses = computed(() => (VariantClassesTable[props.variant || 'primary']))
+const variantClasses = computed(() => {
+  return (VariantClassesTable[props.variant || 'primary']).split(' ').filter((css) => {
+    return !(css === 'hocus-default' && props.disabled)
```

Comment:
I am not a huge fan of having to solve this with string manipulation, but I think it is fine for this use case.  My only request is separate out the new logic with the filter and either put a nice comment on it or wrap it in a function with a descriptive name.  

Also, this solution does not fix the `secondary` button since it uses a different class `hocus-secondary`.  Filter that one out as well.

Reply:
Addressed in [52feb4b](https://github.com/cypress-io/cypress/pull/25234/commits/52feb4bb9e19e34910da6de2cc730478aa1cfb68)